pkgs/development/libraries/webkitgtk/default.nix

```diff
@@ -93,6 +107,14 @@ stdenv.mkDerivation rec {
       src = ./fdo-backend-path.patch;
       wpebackend_fdo = libwpe-fdo;
     })
+  ] ++ [
+    # https://bugs.webkit.org/show_bug.cgi?id=232934
+    ./fix-build-without-opengl-or-es.patch
+    # https://bugs.webkit.org/show_bug.cgi?id=126433
+    ./fix-conflicting-types-on-darwin.patch
+    # non upstreamed patches
+    ./fix-wtf-errors-on-darwin.patch
```

Comment:
I see, these are mostly the changes from patch below. I can't remember why I didn't originally upstream those. Can we do it now?

Reply:
Patch 126433 was actually merged and then reverted.

I'm afraid I don't have time to upstream them these days. I'm welcome if anyone else does it before me.

Reply:
The patch is now upstreamed.